README.md

```diff
@@ -12,21 +12,25 @@ This macro is intended to be used within a salesforce dbt project model. To leve
 ```yml
 packages:
   - package: fivetran/salesforce_formula_utils
-    version: [">=0.3.0", "<0.4.0"]
+    version: [">=0.4.0", "<0.5.0"]
 ```
 > **Note**: In order to use the macros included in this package you will need to have a properly configured source package with a source named `salesforce`. To see an example of a properly configured Salesforce source yml you can reference [integration_tests](integration_tests/models/src_fivetran_formula.yml). You are also welcome to copy/paste this source configuration into your dbt root project and modify for your Salesforce use case.
 
-### Model Creation
-Once the package is added, you may use the macro within your salesforce models. To do so you will create a new file in your models folder and name it (`your_table_name_here`.sql). Then add the below snippet into the file. You will then update the `your_table_name_here` argument to be the table for which you are generating the model:
+### Model Creation: All relevant formula fields
+If you would like your model to generate all the formula fields related to your source table then you may create a new file in your models folder and name it (`your_table_name_here`.sql). You will then add the below snippet into the file. Finally, update the `source_table` argument to be the source table name for which you are generating the model:
 ```sql
-{{ salesforce_formula_utils.sfdc_formula_view('your_source_table_name_here') }}
+{{ salesforce_formula_utils.sfdc_formula_view(source_table='your_source_table_name_here') }}
+```
+### Model Creation: Only include specified formula fields
+If you would like your model to generate only a specified subset of your formula fields related to your source table then you may create a new file in your models folder and name it (`your_table_name_here`.sql). You will then add the below snippet into the file. Finally, update the `source_table` argument to be the source table name for which you are generating the model and update the `fields_to_include` argument to contain all the fields (in **lowercase**) from your source that you would like to be included in the final output:
+```sql
+{{ salesforce_formula_utils.sfdc_formula_view(source_table='your_source_table_name_here', fields_to_include=('i_want_this_field','also_this_one','maybe_a_third_as_well','lets_add_more')) }}
```

Comment:
j curious - is jinja is ok with providing lists using parentheses instead of `[]`?

Reply:
It does! I ended up going the route of `()` because in the get_column_values macro I want to add the `fields_to_include` within the `where field in ()` If we used the `[]` we would have had to include extra logic to replace the square brackets and I felt the parenthesis would result in a cleaner end product.

Reply:
Maybe make a note in the readme on using the parenthesis